network-api/networkapi/buyersguide/management/commands/migrate_products.py

```diff
@@ -187,10 +187,13 @@ def handle(self, *args, **options):
                 new_product_page.get_or_create_votes()
                 # Use .to_dict() to pull out the old aggregated votes
                 product_dict = product.to_dict()
-                if hasattr(product_dict, 'votes'):
-                    votes = product_dict['votes']['creepiness']['vote_breakdown']
+                votes = product_dict.get('votes', None)
+                if votes:
+                    print(votes)
```

Comment:
Let's use `self.debug_print()` here so we don't spam the production or staging logs.

Reply:
```suggestion
                    self.debug_print(votes)
```